src/view3d.cpp

```diff
@@ -78,7 +91,7 @@ View3D::View3D(GeometryCollection* geometries, QWidget *parent)
 
     m_incrementalFrameTimer = new QTimer(this);
     m_incrementalFrameTimer->setSingleShot(false);
-    connect(m_incrementalFrameTimer, SIGNAL(timeout()), this, SLOT(updateGL()));
+    // connect(m_incrementalFrameTimer, SIGNAL(timeout()), this, SLOT(updateGL()));
```

Comment:
This should be put back as it was I suppose.

The incremental frame timer allows point clouds which are too large to be rendered in successive frames.  The visual effect is an unfortunate flickering, but it does give you a view of the whole dataset which can be quite important for technical work.